WordPress/Classes/Utility/WPMapFilterReduce.h

```diff
@@ -0,0 +1,37 @@
+#import <Foundation/Foundation.h>
+
+typedef id (^WPMapBlock)(id obj);
+typedef BOOL (^WPFilterBlock)(id obj);
+typedef id (^WPReduceBlock)(id accumulator, id obj);
```

Comment:
Acknowledged. In that case, we may wanna revisit and wire the Generics support, the type of this fellow should match the one that's contained in the collection (though.. i'm not 100% sure how the syntax should look like!)